kavenegar.d.ts

```diff
@@ -0,0 +1,51 @@
+declare namespace kavenegar {
+    interface IOptions {
+        host?: string;
+        version?: string;
+        /**
+         * apikey :
+         * Get apikey form the user panel Or talk to the technical department for guidance
+         */
+        apikey: string;
+    }
+
+    interface IEntrie {
+        messageid: 8792343,
+        message: string,
+        status: number,
+        statustext: string,
+        sender: string,
+        receptor: string,
```

Comment:
I think you should change this to, 
```typescript 
receptor: string[] || string
```
because in some cases we can also pass array of numbers and send to multiple phone numbers, this is also not a good idea because it takes so long even for small number of receptor items